README.md

```diff
@@ -88,6 +88,41 @@ Meanwhile, [Plenoptic Dataset](https://github.com/facebookresearch/Neural_3D_Vid
 |     ├── ...
 ```
 
+**For multipleviews scenes:**
+The dataset privided in [Ballett5](https://www.swisstransfer.com/d/b3f1ebc8-def6-428f-924a-9a132a86a2f9) is used.If you want to train your own dataset of multipleviews scenes,you can orginize your dataset like Ballett5 as follows:
+```
+├── data
+│   | cam01
+|     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | cam02
+│     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | ...
+```
+After that,you can use the  `multipleviewprogress.sh` I provided to generate related data of poses and pointcloud.You need to 
+ensure that the data folder is orginized as follows after running:
+```
+├── data
+│   | cam01
+|     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | cam02
+│     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | ...
+│   | sparse_
+│     ├── cameras.bin
+│     ├── images.bin
+│   | points3D_multipleview.ply
+│   | poses_bounds_multipleview.npy
```

Comment:
And the ` ./data/$workdir/poses_bounds_multipleview.npy` maybe can be laoded by `dynerf`'s dataloader? Do we need another independent dataloader to load the dataset?